web/src/beta/components/RangeSlider/index.tsx

```diff
@@ -0,0 +1,70 @@
+import RCSlider from "rc-slider";
+import React, { ComponentProps } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+import "rc-slider/assets/index.css";
+
+const RangeSliderWithTooltip = RCSlider.createSliderWithTooltip(RCSlider.Range);
+
+export type Props = {
+  min?: number;
+  max?: number;
+} & ComponentProps<typeof RangeSliderWithTooltip>;
+
+const RangeSlider: React.FC<Props> = ({ ...props }) => {
+  const calculatedStep = props.step ? props.step : props.max ? props.max / 10 : 0.1;
```

Comment:
Done

Reply:
Same here

Reply:
Hi @nina992 i notice you updated this part with `!!min && !!max`. Maybe it's better to check value like `min !== undefined` since `!!0` will be `false` and `0` could be widely used for min.

Reply:
@airslice  noted , thank you , fixed